integration-tests/tests/specs/features/authenticated.spec.ts

```diff
@@ -2,8 +2,12 @@ import { expect } from '@playwright/test';
 import { test } from '../../fixtures/auth.fixture';
 
 test.describe('Basic test of authenticated fixture', () => {
-    test('authenticated fixture runs and lands on user page', async ({ pageWithACreatedUser }) => {
-        await expect(pageWithACreatedUser).toHaveURL('/');
-        await expect(pageWithACreatedUser.getByRole('link', { name: 'My account' })).toBeVisible();
+    test('authenticated fixture runs and lands on user page', async ({
+        page,
+        authenticatedUser,
+    }) => {
+        void authenticatedUser;
+        await expect(page).toHaveURL('/');
+        await expect(page.getByRole('link', { name: 'My account' })).toBeVisible();
```

Comment:
We can beef this up now by asserting on the authenticatedUser information - taking full advantage of the new pattern